src/olympia/api/serializers.py

```diff
@@ -40,9 +41,16 @@ def get_fields(self):
                 fields[key] = ESTranslationSerializerField(source=field.source)
         return fields
 
-    def to_native(self, data):
+    def to_internal_value(self, data):
```

Comment:
`to_native` existed and had tests so I assumed it's being used somewhere - wasn't that the whole point behind `self.fake_object`?